wurst/file/SaveLoadData.wurst

```diff
@@ -0,0 +1,82 @@
+package SaveLoadData
+import FileIO
+import SyncSimple
+import ErrorHandling
+
+/* 	Comfort wrapper for player based, synced access to save data.
+	For small data you can use strings, otherwise use ChunkedString.
+
+	Save data for a player:
+	>	myPlayer.saveData("myFileName", "someDataString")
+	>	myPlayer.saveData("myChunkedFileName", someChunkedString)
+
+	Load data for a player:
+	>	myPlayer.loadData("myFileName") (status, data) ->
```

Comment:
I don't see any reason why this should consume a closure - why not just `let status_data = mPlayer.loadData("filename")`?